graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -35,12 +46,12 @@ def rename_schema(ast, renamings):
     belonging to the root/query type will never be renamed.
 
     Args:
-        ast: Document, representing a valid schema that does not contain extensions, input
-             object definitions, mutations, or subscriptions, whose fields of the query type share
-             the same name as the types they query. Not modified by this function
-        renamings: Dict[str, str], mapping original type/field names to renamed type/field names.
-                   Type or query type field names that do not appear in the dict will be unchanged.
-                   Any dict-like object that implements get(key, [default]) may also be used
+        schema_ast: represents a valid schema that does not contain extensions, input object
+                    definitions, mutations, or subscriptions, whose fields of the query type share
+                    the same name as the types they query. Not modified by this function
+        renamings: maps original type/field names to renamed type/field names. Type or query type
+                   field names that do not appear in the dict will be unchanged. Any dict-like
+                   object that implements get(key, [default]) may also be used
```

Comment:
The type hint of `Dict[str, str]` will actually prohibit this "dict-like" bit from being true. I think you are looking for `Mapping` instead of `Dict` here: https://docs.python.org/3/library/typing.html#typing.Mapping